pkg/pillar/cmd/msrv/pubsub.go

```diff
@@ -0,0 +1,506 @@
+package msrv
```

Comment:
Please add copyright and license lines to new files like this.

Reply:
Done